docs/reference/scripting/security.asciidoc

```diff
@@ -8,6 +8,52 @@ all software has bugs and it is important to minimize the risk of failure in
 any security layer. Find below rules of thumb for how to keep Elasticsearch
 from being a vulnerability.
 
+[discrete]
+[[prefer-params]]
```

Comment:
This is nice but does not go in the security section.  Perhaps in how to write scripts?

Reply:
Agreed. I'll incorporate it into the section for how to write scripts and build that narrative out more.